### PR TITLE
17982 - Unable to obtain or confirm information.

### PR DESCRIPTION
**Pull request by hfekete**
*Issue:17982 *https://github.com/bcgov/entity/issues/17982 

*Description of changes:*
- added message box component (alert, warning, info ...)
- added section in add new individual that is dealing with "Unable to obtain or confirm information" data

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-63-ew8sa3uw.web.app

File: btr-web/btr-main-app/components/individual-person/AddNew.vue [hunk 1]
```diff
@@ -169,6 +169,39 @@
...
+        <p class="py-3">
+          {{ $t('labels.unableToObtainOrConfirmInformation.description') }}
+        </p>
+        <UTextarea
+          v-model="isUnableToObtainOrConfirmInformationDetails"
+          :placeholder="$t('labels.unableToObtainOrConfirmInformation.textAreaPlaceholder')"
+          class="py-2 w-full"
+          variant="bcGov"
+          data-cy="isUnableToObtainOrConfirmInformationTextArea"
+          @change="isUnableToObtainOrConfirmInformationDetailsChange"
```
Comment (kialj876) [inline btr-web/btr-main-app/components/individual-person/AddNew.vue:193]: I think to make it draggable there's a resize prop

File: btr-web/btr-main-app/components/individual-person/AddNew.vue [hunk 2]
```diff
@@ -169,6 +169,39 @@
...
+        />
```
Comment (kialj876) [inline btr-web/btr-main-app/components/individual-person/AddNew.vue:194]: can you add the character count underneath like the design here

File: btr-web/btr-main-app/components/individual-person/AddNew.vue [hunk 3]
```diff
@@ -271,6 +304,14 @@ const taxInfoModel = computed({
 
 // tax residency
 const isTaxResident: Ref<string | undefined> = ref(undefined)
+
+const isUnableToObtainOrConfirmInformation = ref(false)
+const isUnableToObtainOrConfirmInformationDetails = ref('')
+const isUnableToObtainOrConfirmInformationDetailsChange = () => {
```
Comment (kialj876) [inline btr-web/btr-main-app/components/individual-person/AddNew.vue:310]: if you convert this to a watcher on `isUnableToObtainOrConfirmInformationDetails` it'll have the desired effect. Double check with Scott, but I think it should be getting checked / unchecked in real time not blur?

**Review by kialj876 — COMMENTED** [review]
Hey Hrvoje, nice work. Just a few minor comments

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-63-ew8sa3uw.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-63-ew8sa3uw.web.app

**Review by patrickpeinanw — APPROVED** [review]

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-btr-dev--pr-63-ew8sa3uw.web.app

**Pull request merged**